controllers/druid/handler.go

```diff
@@ -190,6 +192,19 @@ func deployDruidCluster(sdk client.Client, m *v1alpha1.Druid) error {
 				}
 			}
 		} else {
+
+			//	scalePVCForSTS to be only called only if volumeExpansion is supported by the storage class.
+			//  Ignore for the first iteration ie cluster creation, else get sts shall unnecessary log errors.
+
+			if m.Generation > 1 {
+				if isVolumeExpansionEnabled(sdk, m, &nodeSpec) {
+					err := scalePVCForSts(sdk, &nodeSpec, nodeSpecUniqueStr, m)
+					if err != nil {
+						return err
+					}
+				}
+			}
```

Comment:
Ill add a flag for this, users should be aware of this feature.

Reply:
is the flag added now ?